packages/react-composites/src/composites/CallComposite/pages/UnsupportedBrowser.tsx

```diff
@@ -34,9 +30,7 @@ export const UnsupportedBrowserPage = (props: UnsupportedBrowserPageProps): JSX.
   /* @conditional-compile-remove(unsupported-browser) */
   const unsupportedBrowserVersionStrings = locale.component.strings.UnsupportedBrowserVersion;
 
-  /* @conditional-compile-remove(unsupported-browser) */
   let pageElement: JSX.Element | undefined;
-  /* @conditional-compile-remove(unsupported-browser) */
   if (!environmentInfo?.isSupportedBrowser) {
     pageElement = (
       <UnsupportedBrowser onTroubleshootingClick={onTroubleshootingClick} strings={unsupportedBrowserStrings} />
```

Comment:
Doesn't this complain that `unsupportedBrowserStrings` is undefined?

I expected that you'll need a `getStringsTrampoline()` function that returns empty strings for this to work.